src/form/validators.ts

```diff
@@ -168,9 +174,10 @@ export const nonEmptyMultiPolygonValidatorGenerator =
   };
 
 export const eitherMultiPolygonOrRequiredValidatorGenerator =
-  (comparisonValue?: MultiPolygon | null, customError?: string) =>
-  (value?: string): string | null => {
-    if (comparisonValue) return null;
+  (customError?: string) =>
+  (value: string, values: any): string | null => {
```

Comment:
Wondering if values.search is typed somewhere in the project?

Reply:
Its redux-forms data that's passed in by the redux-form Field component on validate. I can try add some in code comment to make it clearer